train.py

```diff
@@ -105,7 +105,7 @@ def train(args):
                                     'weighted_rmse', 'weighted_mae', 'rmse', 'mse'])
     
     # Epoch (Training) loop
-    for epoch in tqdm(range(args.epochs)):
```

Comment:
why are are you removing tqdm here?

Reply:
I first added it as I wanted an outer progress bar, however, it did not work.